src/browser/main.coffee

```diff
@@ -77,14 +77,9 @@ setupAtomHome = ->
     atomHome = fs.realpathSync(atomHome)
   process.env.ATOM_HOME = atomHome
 
-setupCoffeeCache = ->
-  CoffeeCache = require 'coffee-cash'
-  cacheDir = path.join(process.env.ATOM_HOME, 'compile-cache')
-  # Use separate compile cache when sudo'ing as root to avoid permission issues
-  if process.env.USER is 'root' and process.env.SUDO_USER and process.env.SUDO_USER isnt process.env.USER
-    cacheDir = path.join(cacheDir, 'root')
-  CoffeeCache.setCacheDirectory(path.join(cacheDir, 'coffee'))
-  CoffeeCache.register()
+setupCompileCache = ->
+  compileCache = require('../compile-cache')
```

Comment:
> 14ms presumably on the render side

I would expect close to zero difference on the render side, because `babel.js`, `typescript.js`, `coffee-stack`, `coffee-cash` were all previously eager-loaded by `index.js`. This adds `source-map-support` but removes 2 other npms (`coffee-stack` and `coffee-cash`).